linera-chain/src/data_types.rs

```diff
@@ -285,9 +288,8 @@ pub enum Medium {
 #[cfg_attr(with_testing, derive(Eq, PartialEq))]
 pub struct BlockProposal {
     pub content: ProposalContent,
-    pub owner: Owner,
```

Comment:
Not sure if we want to remove that, or have the owner explicitly in the proposal.

I actually like the idea, since it removes a way in which a proposal could be invalid (and which could be missed on validation). But it should be added to the PR description.

(Originally the idea was that since we are using `Owner` in most user-facing places now, it is more self-explanatory if the user is an explicit field.)

Reply:
I can extract it to a separate PR if you prefer?

Reply:
This change was reverted in [Revert "Remove Owner from BlockProposal."](https://github.com/linera-io/linera-protocol/pull/3366/commits/c1bb7b452e7501e326dfecc2e8f0b30cfb27b606). I will create a separate PR for it.